packages/cli-main/src/index.ts

```diff
@@ -39,21 +39,24 @@ async function runCLI() {
 
 const reportError = async (errorToReport: Error): Promise<Error> => {
   await analytics.reportEvent({errorMessage: errorToReport.message})
+  if (settings.debug || !kitError.shouldReport(errorToReport)) return errorToReport
 
-  if (!settings.debug && kitError.shouldReport(errorToReport)) {
-    let mappedError: Error
-    // eslint-disable-next-line no-prototype-builtins
-    if (Object.prototype.isPrototypeOf(errorToReport)) {
-      const mappedError = Object.assign(Object.create(errorToReport), {})
-      if (mappedError.stack) mappedError.stack = mappedError.stack.replace(new RegExp('file:///', 'g'), '/')
-    } else {
-      mappedError = errorToReport
-    }
-    await new Promise((resolve, reject) => {
-      Bugsnag.notify(mappedError, undefined, resolve)
-    })
+  let mappedError: Error
+
+  // eslint-disable-next-line no-prototype-builtins
+  if (Error.prototype.isPrototypeOf(errorToReport)) {
+    mappedError = new Error(errorToReport.message)
+    if (errorToReport.stack) mappedError.stack = errorToReport.stack.replace(new RegExp('file:///', 'g'), '/')
```

Comment:
Do Windows stacktraces have `file:///` as prefix? [Here's](https://app.bugsnag.com/shopify/cli/errors/62bf091dacaaaf00089a6652?event_id=62bf091d009557501f540000&i=sk&m=nw) an example of a Windows error. What about using `fileURLToPath` from the `node:url` module to standardize the path?

Reply:
good catch, i'll try that

Reply:
So after some investigation, node does add the `file:///` prefix even in windows:
![IMG_0555AD2A135D-1](https://user-images.githubusercontent.com/3757193/177149995-6c74eaa6-c2b5-498c-9e4a-c7689c24427f.jpeg)

I've update the regex to take that pattern into account for windows.